tools/dep_updaters/README.md

```diff
@@ -35,3 +35,32 @@ been created with the changes), do the following:
    example).
 
 [`c61870c`]: https://github.com/nodejs/node/commit/c61870c376e2f5b0dbaa939972c46745e21cdbdd
+
+## OpenSSL
+
+The `update-openssl.sh` script automates the steps described in
+[`maintaining-openssl.md`][]. The main difference is that the script downloads
+the release tarball from GitHub, instead of cloning the repo and using that as
+the source code. This is useful since the release tarball does not include
+development-specific files and directories (e.g the `.github` folder).
+
+For example, in order to update to version `3.0.7+quic1`, the following command
+can be run:
+
+```bash
+./tools/dep_updaters/update-openssl.sh 3.0.7+quic1
+```
+
+Once the script has run (either manually, or by CI in which case a PR will have
+been created with the changes), do the following:
+
+1. Check the `CHANGES.md` file in the [repo](https://github.com/quictls/openssl)
+   for things that might require changes in Node.js.
+2. Check the diffs to ensure the changes are right. Even if there are no changes
+   in the source, `buildinf.h` files will be updated because they have timestamp
```

Comment:
Not related to this PR, but we should add a task somwhere to see if we can address this. Having it be more repeatable would be good.